src/main/java/frc/utils/joysticks/SmartJoystick.java

```diff
@@ -159,10 +152,7 @@ public double getAxisValue(Axis axis) {
     }
 
     private double deadzone(double power) {
-        if (Math.abs(power) < deadzone) {
-            return 0;
-        }
-        return (Math.abs(power) - deadzone) / (1 - deadzone) * Math.signum(power);
+        return Math.abs(power) < deadzone ? 0 : power;
```

Comment:
instead of using the previous equation, just using the power would make it hard for slight movements of the joystick